README.md

```diff
@@ -55,6 +55,26 @@ zpm update
 
 ```
 
+To remove plugin:
+
+```sh
+zpm remove user/plugin
+zpm remove /path/to/local/plugin
+```
+
+To uninstall plugin:
+
+```sh
+zpm uninstall user/plugin
+zpm uninstall /path/to/local/plugin
+```
+
+To make a copy of your config
+
+```sh
+zpm save > .zpmrc
```

Comment:
We probably will need to explain the workflow with an example. Ex:

Install some plugins:

    zpm "some/plugin"
    zpm "other/plugin"

Then you can persist your current configuration to be able to commit it into a dotfile repository:

    zpm save > ~/dotfiles/.zpmrc

Next time you'll only need to install them like so:

    source ~/dotfiles/.zpmrc